pkg/promotion/runner/builtin/schemas/set-freight-alias-config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "SetFreightAliasConfig",
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["freightID","newAlias"],
+  "properties": {
+    "freightID": {
```

Comment:
I forgot we internally call this "Freight name", even when it looks like an "ID". We should probably rename this to `freightName`.